python/lsst/analysis/tools/actions/plot/plotUtils.py

```diff
@@ -602,6 +602,7 @@ def plotProjectionWithBinning(
     isSorted=False,
     vmin=None,
     vmax=None,
+    addExtremeScatter=True,
```

Comment:
`addExtremeScatter` is not my favorite name for this, I think of it as something like "show outlier points" or "display large outliers" or some other combination with a more display-oriented verb.  ("add" sounds like it's changing the values)

Reply:
ok - proposed name change to "showExtremeOutliers"